packages/web-client/app/components/card-pay/layer-one-connect-card/index.hbs

```diff
@@ -7,6 +7,11 @@
   ...attributes
 >
   {{#if this.connectedWalletProvider}}
+    <Listener
+      @emitter={{this.layer1Network}}
+      @event="disconnect"
+      @action={{optional @onIncomplete}}
+    />
```

Comment:
An example of use of the `Listener` component:
- `@emitter` is the service/object to listen to
- `@event` is the name of the event
- `@action` is the triggered callback